app/eg021_phone_authentication.py

```diff
@@ -0,0 +1,159 @@
+                        document_id="1",  # a label used to reference the doc
+                        file_extension="pdf",  # many different document types are accepted
+                        name="Lorem"  # can be different from actual file name
+            )
+            envelope_definition.documents = [document1]
+            envelope_definition.status = args['envelope_args']['status']
+            signer1 = Signer(
+                email= args["envelope_args"]["signer_email"], # represents your {signer_email}
+                name= args["envelope_args"]["signer_name"], # represents your {signer_name}
+                phone_authentication= { "senderProvidedNumbers": ["415-555-1212"]},
```

Comment:
same comment, but if we have to hardcode let's have it in a variable that's easy to update